tests/fuzzer.c

```diff
@@ -1286,6 +1286,64 @@ static int basicUnitTests(U32 const seed, double compressibility)
     }
     DISPLAYLEVEL(3, "OK \n");
 
+    DISPLAYLEVEL(3, "test%3i : Check block splitter with 64K literal length : ", testNb++);
+    {   ZSTD_CCtx* cctx = ZSTD_createCCtx();
+        size_t const srcSize = 256 * 1024;
+        U32 const compressibleLenU32 = 32 * 1024 / 4;
+        U32 const blockSizeU32 = 128 * 1024 / 4;
+        U32 const litLenU32 = 64 * 1024 / 4;
+        U32* data = (U32*)malloc(srcSize);
+        size_t dSize;
+
+        if (data == NULL || cctx == NULL) goto _output_error;
+
+        /* Generate data without any matches */
+        RDG_genBuffer(data, srcSize, 0.0, 0.01, 2654435761U);
```

Comment:
Is this sensitive to the seed `2654435761U`, in the sense that future implementations of `RDG_genBuffer` may not generate exactly zero matches for this seed? If so, perhaps we should use a [DeBruijn sequence generator](https://gist.github.com/embg/4f445b51c362ed1dbd6267b13d1e4f11)?

Reply:
@embg yeah it would make sense to use that, and to move this to a CLI test with a golden file. So changes to `RDG_genBuffer()` don't affect the test.